### PR TITLE
[feature1] First feature was implemented

### PR DESCRIPTION
**Pull request by mikegithubber**
Following changes were made:

- new-feature.txt file was added
- feature1 folder was created

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```
Comment (bstashchuk) [inline feature1/one-more-file.txt]: It's really good that you have decided to add this line of code

Comment (mikegithubber) [reply feature1/one-more-file.txt]: Thanks for this comment, Bogdan!

**Review by bstashchuk — APPROVED** [review]
Good job, Mike!

**Review by kosemMG — APPROVED** [review]
Thanks for the great course, Bogdan!

**Comment by sulaber** [comment]
Very good course Bogdan, Thank you.

**Comment by Dev-Myk** [comment]
Awesome course!

**Comment by sameergithub5** [comment]
Its actually awesome in dept knowledge. It would be better if you could ask more questions or keep cert exam in the end.

**Review by mdct110 — COMMENTED** [review]
Good job

**Comment by calebfoster** [comment]
I can't get this to run

**Comment by shrinidhijahagirdar** [comment]
@bstashchuk _/\_

File: new-feature.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This file was created in the feature1 branch
```
Comment (shrinidhijahagirdar) [inline new-feature.txt:1]: I appreciate this line!

File: feature1/one-more-file.txt (hunk 1 above)
Comment (helloabie) [inline feature1/one-more-file.txt:1]: very nice line, Mike!

File: feature1/one-more-file.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+One more change in feature1 branch
```
Comment (isoboye24) [inline feature1/one-more-file.txt:3]: It is good that you have decided to add this line of the code :)

File: feature1/one-more-file.txt (hunk 1 above)
Comment (DevelopCarlosMD) [inline feature1/one-more-file.txt:1]: Thanks for the course a little late

**Comment by npworkdev** [comment]
Can't wait to see what "Squash and Merge" and "Rebase and Merge" do. Keep up the great teaching.

File: feature1/one-more-file.txt (hunk 1 above)
Comment (blmille1) [reply feature1/one-more-file.txt:1]: @bstashchuk I'm in the Brave Browser and I am able to send you a tip, but it says that you've not yet verified.  Have you considered doing that? It's a way for us to tip the content creators whom we love.  Thank you for this wonderful course!  I really love it so far!

Comment (blmille1) [reply feature1/one-more-file.txt:1]: image
See the Tip button?

Comment (Tristero49) [inline feature1/one-more-file.txt:1]: This is helpful info!

Comment (alistairreynolds) [inline feature1/one-more-file.txt:1]: More comments!

**Review by alistairreynolds — COMMENTED** [review]
Other comments

**Comment by vivekbprabhu** [comment]
This course is real good. Thanks Bogdan.

File: feature1/one-more-file.txt (hunk 1 above)
Comment (jontiefer) [reply feature1/one-more-file.txt:1]: Hi Bogdan, liking the course.. Just testing out the comments on the pull requests section of the course.

Comment (tomkedem) [reply feature1/one-more-file.txt:1]: I'm sure it's really good that you have decided to add this line code, **just very good**

Comment (DagnyDK) [inline feature1/one-more-file.txt:1]: Outstanding line of code **_indeed!_** sorry about the email you'll get

**Review by ghost — COMMENTED** [review]
Great job, mike

File: feature1/one-more-file.txt (hunk 1 above)
Comment (hema8Codes) [inline feature1/one-more-file.txt:1]: awesome line of code

**Review by EhsanKhalili — COMMENTED** [review]
Great changes Mike ;))))

**Comment by techymaj** [comment]
For glory. For Immortality.

For glory's sake. For vengeance's sake.

Bogdan!!!

**Comment by maineyperfectionist** [comment]
I believe we all like the changes that have been made. Good job!

File: feature1/one-more-file.txt (hunk 1 above)
Comment (maineyperfectionist) [reply feature1/one-more-file.txt:1]: I do, indeed, think that adding this line of code has been very important in the grand scheme of things.

**Comment by xlerach** [comment]
Ola

**Review by xlerach — COMMENTED** [review]
Nice job done

File: feature1/one-more-file.txt (hunk 1 above)
Comment (xlerach) [inline feature1/one-more-file.txt:1]: I liked this

**Review by xlerach — COMMENTED** [review]
Nice job

File: feature1/one-more-file.txt (hunk 1 above)
Comment (nazifullah1) [inline feature1/one-more-file.txt:1]: This is nice that you added this file in the feature1 branch.

**Comment by JaimeSilva** [comment]
Hello :)

File: feature1/one-more-file.txt (hunk 1 above)
Comment (HuseyinBeller) [reply feature1/one-more-file.txt:1]: Great course!

**Comment by HuseyinBeller** [comment]
This is so nice that you added this file in the feature1 branch

**Review by HuseyinBeller — COMMENTED** [review]
Nice!

File: feature1/one-more-file.txt (hunk 1 above)
Comment (HuseyinBeller) [inline feature1/one-more-file.txt:1]: Nice

Comment (sergolego) [reply feature1/one-more-file.txt:1]: Thank a lot Bogdan! Enjoying this course. Best about Git that I've seen so far)

Comment (AnshuBhatiaGit) [reply feature1/one-more-file.txt:1]: This is a great course, bogdan. Really enjoying the detail concepts of Git and Github. You explain it well by focusing on each topic and explaining step by step.

**Comment by kpanko-testuser** [comment]
> [!WARNING]
>  There is a risk of this pull request causing GitHub to crash. Consider creating backup copy of GitHub first.

**Pull request merged**